jck/build.xml

```diff
@@ -246,16 +212,12 @@
 						<echo>=== env.JCK_GIT_REPO is set to ${env.JCK_GIT_REPO} ===</echo>
 						<echo>start staging jck materials</echo>
 						<antcall target="stage_jck_material" inheritall="true" />
-						<echo>start building stf-based jck project</echo>
+						<echo>start building stf, stf jck wrapper and jck itself</echo>
 						<antcall target="dist" inheritall="true" />
-						<echo>start compiling jck natives</echo>
-						<antcall target="compile_jck" inheritall="true" />
 					</then>
 					<else>
-						<echo>start building jck project</echo>
+						<echo>start building stf, stf jck wrapper and jck itself</echo>
```

Comment:
Following two lines are in both if and else block. Instead of having twice, we can move it outside of if-else statement.
```
<echo>start building stf, stf jck wrapper and jck itself</echo>
<antcall target="dist" inheritall="true" />
```

Reply:
Good catch! Updated PR.